aiounifi/models/api.py

```diff
@@ -27,6 +29,16 @@ def full_path(self, site: str, is_unifi_os: bool) -> str:
             return f"/proxy/network/api/s/{site}{self.path}"
         return f"/api/s/{site}{self.path}"
 
+    def prepare_data(self, raw: bytes) -> TypedApiResponse:
+        """Put data, received from the unifi controller, into a TypedApiResponse."""
+        json_data = orjson.loads(raw)
+        return_data: TypedApiResponse = {}
+
+        return_data["meta"] = json_data.get("meta", None)
+        return_data["data"] = json_data.get("data", None)
+
+        return return_data
```

Comment:
Lets consider this method only supporting v1 responses and do an override in the traffic rules api request child
```suggestion
    def prepare_data(self, raw: bytes) -> TypedApiResponse:
        """Put data, received from the unifi controller, into a TypedApiResponse."""
        data: TypedApiResponse = orjson.loads(raw)
        return data
```

Reply:
That's when you're in too deep... I did my reasoning the wrong way round. Since the structure of the trafficrules-response does not contain a meta- and data-key this wasn't working, so I overcomplicated things at the higher level.

Reply:
Well you got to a point where everything worked right. Bow we're just trying to fit everything together